ttnn/ttnn/__init__.py

```diff
@@ -304,7 +304,7 @@ def prelu(*args, **kwargs):  # Alias for leaky_relu. TODO(#8544): implement PReL
 )
 
 from ttnn.operations.conv2d import Conv2d, Conv2dConfig, get_conv_output_dim, get_conv_padded_input_shape_and_mem_config
-from ttnn.operations.pool import TTPyMaxPool, max_pool2d, max_pool2d_legacy, MaxPool2d, avg_pool2d
+from ttnn.operations.pool import avg_pool2d
```

Comment:
are you sure to remove `max_pool2d` from here too?

Reply:
yes, thats the old version